### ISSUE TITLE
How to run Python tests?

### ISSUE THREAD

**Issue by svpino** [GoogleCloudPlatform/appengine-pipelines#27]
I also saw this question on the Google Groups.

I can't find a way to run the tests for Python:

$ cd python/test
$ python pipeline_test.py

WARNING:root:Could not load Pipeline API. Will fix path for testing. ImportError: No module named yaml
Traceback (most recent call last):
  File "pipeline_test.py", line 40, in <module>
    from pipeline import common
  File "../src/pipeline/__init__.py", line 49, in <module>
    from pipeline import *
  File "../src/pipeline/pipeline.py", line 43, in <module>
    from google.appengine.api import files
  File "/usr/local/google_appengine/google/appengine/api/files/__init__.py", line 30, in <module>
    import blobstore
  File "/usr/local/google_appengine/google/appengine/api/files/blobstore.py", line 39, in <module>
    from google.appengine.api.files import file as files
ImportError: cannot import name file
"""

**Comment by sadovnychyi** [comment]
Same here, cannot make a pull request without running the tests.

**Comment by svpino** [comment]
Alright so I figure this out.

First of all, the current test cases are using `old_dev_appserver` which is deprecated and removed from the platform. I submitted a pull request to resolve the issue.

@sadovnychyi, if you want to run the tests, do the following:

1) Download the project.
2) Apply the changes I submitted in PullRequest #28 
3) From the command line, execute the following: 
`
    $ PYTHONPATH="/usr/local/google_appengine"
    $ export PYTHONPATH
`
(This is assuming that `/usr/local/google_appengine` is the path of appengine in your computer.)
4) Go inside the `python/test` folder
5) Run the tests: `python pipeline_test.py`

Let me know if it doesn't work.
